src/Components/Atoms/FAChip/FAChip.interface.tsx

```diff
@@ -0,0 +1,8 @@
+import { ChipProps } from "@mui/material";
+
+export interface IFAChip extends ChipProps {
+  label?: string;
+  testID: `chip-${string}`;
+  avatar?: any;
+  deleteable?: boolean;
```

Comment:
Should we remove `deletable` props? if the purpose of these props is to change the icon, I think it's better to use the default icon of the chip component.

Reply:
the very purpose of the deletable prop IS NOT to change the icon rather it is to STOP the delete icon from appearing